lifecycle/export.go

```diff
@@ -0,0 +1,149 @@
+package lifecycle
+
+import (
+	"encoding/json"
+	artUtils "github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	artServices "github.com/jfrog/jfrog-client-go/artifactory/services"
+	"github.com/jfrog/jfrog-client-go/artifactory/services/utils"
+	clientConfig "github.com/jfrog/jfrog-client-go/config"
+	"github.com/jfrog/jfrog-client-go/lifecycle/services"
+	utils2 "github.com/jfrog/jfrog-client-go/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"strings"
+)
+
+type ReleaseBundleExportCommand struct {
+	releaseBundleCmd
+	modifications          *services.Modifications
+	downloadConfigurations *artUtils.DownloadConfiguration
+}
+
+func (rbe *ReleaseBundleExportCommand) Run() (err error) {
+	if err = validateArtifactoryVersionSupported(rbe.serverDetails); err != nil {
+		return
+	}
+	servicesManager, rbDetails, queryParams, err := rbe.getPrerequisites()
+	if err != nil {
+		log.Debug("Failed getting prerequisites for exporting command, error: ", err.Error())
+		return
```

Comment:
Debug logs can be missed. If you want to add a message to the error, use:

```suggestion
		return errorutils.CheckErrorf("failed getting prerequisites for exporting command, error: '%s'", err.Error())
```